# custom message for fisheye point reprojection to have also covariance

std_msgs/Header header

geometry_msgs/PoseWithCovariance[] poses_with_cov_matrices
